### ISSUE TITLE
Not able to turn off progress bars

### ISSUE THREAD

**Issue by twalen** [nalepae/pandarallel#174]
## General
- **Operating System**: Macos 11.6.4
- **Python version**: 3.7.12
- **Pandas version**: 1.3.3 
- **Pandarallel version**: 1.5.6

## Acknowledgement
- [X] I am **NOT** using Windows without WSL (Windows Subsytem for Linux)
- [X] My issue is **NOT** present when using `pandas` without alone (without `pandarallel`)

### Observed behavior
It is not possible to silent progress bar using `pandarallel.initialize(progress_bar=False)` in versions 1.5.6 and 1.5.7.
As a result progress_bar is rendered in the output.
This bug is not present in version 1.5.5.

### Expected behavior
With `progress_bar=False` the progress bar should not be rendered.
*Write here the expected behavior*

## Minimal but working code sample to ease bug fix for `pandarallel` team

```
import pandas as pd
from pandarallel import pandarallel

pandarallel.initialize(progress_bar=False, verbose=0)
pd.DataFrame({"a": range(10**4)}).parallel_apply(lambda x: x, axis=1)
```

**Comment by nalepae** [comment]
Thank you for your bug report.
I do reproduce your issue on Terminal only (everything seems OK on Jupyter Notebook / Jupyter Lab).

**Comment by nalepae** [comment]
Fixed in `pandarallel v1.5.8`.
